.github/workflows/update-plugin-list.yml

```
@@ -0,0 +1,32 @@
+name: Update Plugin List
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

For those of us who can't remember cron syntax even after using it a thousand times :)

```suggestion
    # Run daily at midnight.
    - cron: '0 0 * * *'
```